lib/apiservers/service/restapi/handlers/vch_cert_get.go

```diff
@@ -35,21 +35,23 @@ type VCHCertGet struct{}
 type VCHDatacenterCertGet struct{}
 
 func (h *VCHCertGet) Handle(params operations.GetTargetTargetVchVchIDCertificateParams, principal interface{}) middleware.Responder {
-	d, err := buildData(params.HTTPRequest.Context(),
-		url.URL{Host: params.Target},
-		principal.(Credentials).user,
-		principal.(Credentials).pass,
-		params.Thumbprint,
-		nil,
-		nil)
+	op := trace.NewOperation(params.HTTPRequest.Context(), "VCHCertGet: %s", params.VchID)
```

Comment:
Can you ensure that anywhere that you're introducing an operation, it replaces all occurrences of `params.HTTPRequest.Context()` (not just the first)?

Reply:
Fixed, thanks